model/src/main/java/org/projectnessie/api/ContentsApi.java

```diff
@@ -84,70 +82,4 @@ public MultiGetContentsResponse getMultipleContents(
           MultiGetContentsRequest request)
       throws NessieNotFoundException;
 
-  /**
-   * create/update an object on a specific ref.
-   */
-  @POST
```

Comment:
remove two endpoints that are unused by clients (hms, iceberg, delta) and are really just helper methods for multi-commit. Thoughts?